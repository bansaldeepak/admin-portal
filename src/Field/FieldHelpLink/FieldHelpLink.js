import React from "react";
import FormHelperText from "@material-ui/core/FormHelperText";

const FieldHelpLink = (props) => {
  const { helplink, name, fieldValues } = props;

  return helplink !== undefined ? (
    <FormHelperText id={`${name}-help-link`} style={{ textAlign: "right" }}>
      {typeof helplink === "function" ? helplink(fieldValues) : helplink}
    </FormHelperText>
  ) : null;
};

export default FieldHelpLink;
