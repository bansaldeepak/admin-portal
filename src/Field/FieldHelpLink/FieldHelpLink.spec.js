import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallowToJson } from "enzyme-to-json";
import FieldHelpLink from "./FieldHelpLink";

configure({ adapter: new Adapter() });

describe("FieldHelpLink", () => {
  it("Render FieldHelpLink with default options", () => {
    const component = shallow(<FieldHelpLink error="FieldHelpLink" />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
