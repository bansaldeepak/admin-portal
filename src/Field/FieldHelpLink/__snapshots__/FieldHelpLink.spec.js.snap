// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldHelpLink Render FieldHelpLink with default options 1`] = `""`;
